spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Graph.scala

```diff
@@ -159,7 +159,7 @@ abstract class Graph[T: ClassTag](
   protected val forwardNodes = forwardGraph.DFS.toArray
 
   modules.appendAll(
-    forwardGraph.topologySort
+    forwardGraph.DFS.toArray
       // todo: convert control dep node to edge
       .filterNot(_.element.isInstanceOf[ControlDependency[T]]).reverse
```

Comment:
Maybe we do not need to reverse, if we do not care about the order.

Reply:
fixed

Reply:
Let's keep it. Some unit test compares graph parameters and sequential parameters. Remove it will cause the sequence of the parameters are not same...